lib/swagger-explorer.ts

```diff
@@ -205,10 +205,17 @@ export class SwaggerExplorer {
     return {
       method: RequestMethod[requestMethod].toLowerCase(),
       path: fullPath === '' ? '/' : fullPath,
-      operationId: method.name
+      operationId: this.getOperationId(instance, method)
     };
   }
 
+  private getOperationId(instance: object, method: Function): string {
+    if (instance.constructor) {
+      return `${instance.constructor.name}_${method.name}`;
```

Comment:
I'm not sure if we should protect ourselves against code that shouldn't appear in any codebase since it simply violates too many rules (+ probability of having such a naming clash can be brought down to 0)